src/main/java/org/jitsi/jigasi/SipGatewaySession.java

```diff
@@ -407,6 +407,16 @@ public void peerStateChanged(CallPeerChangeEvent evt)
                 });
             }
         }
+
+        logger.info("JVB conference call IN_PROGRESS "
```

Comment:
I don't understand what this means, even after looking at the source code. `IN_PROGRESS` seems to refer to a `CallState`, but is this necessarily the state of the call? Please, let's improve the log message.

Reply:
Yep, will improve it, It was left like that while juggling with the code.